db/migrations/20221128042517_create_signup_table.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up   
+
+CREATE TABLE users (
+  id int NOT NULL AUTO_INCREMENT,
+  name varchar(50) NOT NULL,
+  email varchar(200) NOT NULL,
+  profile_image varchar(1000) DEFAULT NULL,
+  password varchar(200) NOT NULL,
+  created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at timestamp NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (id)
+);
+
+-- migrate:down
+
+DROP TABLE users;
+
```

Comment:
형진님 ! 의미 없는 줄 바꿈은 지양해 주세요! 아래와 같이 수정해 주시면 감사하겠습니다 !
```suggestion
-- migrate:up   
CREATE TABLE users (
  id int NOT NULL AUTO_INCREMENT,
  name varchar(50) NOT NULL,
  email varchar(200) NOT NULL,
  profile_image varchar(1000) DEFAULT NULL,
  password varchar(200) NOT NULL,
  created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at timestamp NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (id)
);

-- migrate:down
DROP TABLE users;
```